projects/allen-cell-segmenter.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Allen Cell Segmenter
+layout: page
+---
+
+- *Contributor(s)*: Jianxu Chen (@jxchen01), Benjamin Morris (@benjijamorris)
+- *References*: [Project Site](https://www.allencell.org/segmenter.html), [Github](https://github.com/AllenCell/aics-segmentation), [Publication](https://www.biorxiv.org/content/10.1101/491035v2), [Support](https://forum.allencell.org/c/software-code/segmenter/16)
+
+## Purpose
+
+> Describe the problem you are trying to solve by packaging your tool as a napari plugin in 1-2 sentences.
+I should be able to read this alone and communicate the value to someone else.
+
+### Why does this matter to biologists?
+
```

Comment:
The large collection of fluorescently-labeled cell lines from the [Allen Cell Collection](https://www.allencell.org/cell-catalog.html) fosters the development of a large set of image processing workflows that serves as a good resource for biologists to use in many different applications.